package/upgrade/upgrade_node.sh

```diff
@@ -268,18 +262,15 @@ upgrade_os() {
     curl -fL $UPGRADE_REPO_SQUASHFS_IMAGE -o $tmp_rootfs_squashfs
   fi
 
-  tmp_rootfs_mount=$(mktemp -d) 
+  tmp_rootfs_mount=$(mktemp -d -p $HOST_DIR/tmp)
   mount $tmp_rootfs_squashfs $tmp_rootfs_mount
 
-  bash -x $HOST_DIR/usr/sbin/cos upgrade --directory $tmp_rootfs_mount
+  chroot $HOST_DIR cos upgrade --directory ${tmp_rootfs_mount#"$HOST_DIR"}
```

Comment:
We can use `elemental` command to perform the upgrade
```suggestion
  chroot $HOST_DIR elemental upgrade --directory ${tmp_rootfs_mount#"$HOST_DIR"}
```